DTS-STN/passport-status
# Add Test site text on non-prod environments

## [ADO-1897](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1897)

### Description
This PR adds the `Banner` component which is only displayed if the environment variable `NEXT_PUBLIC_ENVIRONMENT` is anything other than `prod`

### What to test for/How to test
1. Pull in branch
2. Add the line `NEXT_PUBLIC_ENVIRONMENT=dev` to your `.env.local`
3. Type `npm run dev`
4. Ensure that the `Banner` component is rendered on `/expectations`
5. Change the value of the environment variable to `prod` and ensure that the component no longer renders
